src/Summoner/Question.hs

```diff
@@ -43,6 +50,27 @@ choose question choices = do
            errorMessage "This wasn't a valid choice."
            choose question choices
 
+chooseYesNo :: Text -- ^ target
+            -> IO a -- ^ action for 'Y' answer
+            -> IO a -- ^ action for 'N' answer
```

Comment:
Nice solution with using `IO` actions as arguments for this function! Really high-level interface :+1: